main.c

```diff
@@ -409,9 +426,15 @@ static void load_image(char *arg, struct swaylock_state *state) {
 
 	// Bash doesn't replace the ~ with $HOME if the output name is supplied
```

Comment:
Could we update this comment to match the reality? (ie. we also need to do a wordexp for the config file)

While you're at it, could replace "bash" with "sh"

Reply:
Done

Reply:
Have you pushed the changes?

Reply:
Oh oops sorry. Apparently, I committed, but never push. Should be pushed now